test/tinycore.sh

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh -e
+#!/usr/bin/env sh
+set -e
```

Comment:
Agree. It's my habit to define this that way. I'll change it back.
Do you prefer ```#!/bin/sh -e``` over
```bash
#!/bin/sh
set -e
```